prost-build/src/lib.rs

```diff
@@ -698,9 +698,10 @@ impl Config {
     ///   Ok(())
     /// }
     /// ```
-    pub fn compile_protos<P>(&mut self, protos: &[P], includes: &[P]) -> Result<()>
+    pub fn compile_protos<P, Pi>(&mut self, protos: &[P], includes: &[Pi]) -> Result<()>
```

Comment:
Can we change these both to be `impl AsRef<Path>` instead of generics on the fn?